build.assets/Dockerfile-centos7

```diff
@@ -198,7 +201,7 @@ RUN yum groupinstall -y 'Development Tools' && \
         autoconf-archive \
         libudev-devel \
         scl-utils \
-        centos-release-scl \
+        centos-release-scl && \
```

Comment:
Is this not being built/tested? This should have been failing, no?

Reply:
Ah, I regressed this in #33886, but it somehow kept working?

Reply:
We had the same issue in the past. We haven't figured out until today why it was working 😆